modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -270,10 +270,23 @@ const EditorManager: EditorManager = {
   },
 
   /**
-   * Overrides the default options for editor instances.
+   * Overrides the default options for editor instances. <code>overrideDefaults</code> replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
```

Comment:
Is it a problem that a lowercased `overrideDefaults` starts a sentence here? Or is that acceptable because it's code?

Reply:
> Is it a problem that a lowercased `overrideDefaults` starts a sentence here? Or is that acceptable because it’s code?

Ooh, look, a can asking to be opened. I wonder if there are worms inside?

Fun fact: modern English capitalisation practice largely derives from the [First Folio](https://en.wikipedia.org/wiki/First_Folio), the edition of Shakespeare’s plays published in 1623, seven years after the playwrights’s death.

It is broadly believed the capitalisation practices used therein (always for the first word of a sentence and only for nouns that named specific things or people; what we now call Proper Nouns) was reflective of Shakespeare’s personal preference, and that he acquired that preference because the community of theatre writers and journalists he was a part of had been pushing for this approach during his working life.

During Shakespeare’s working life (and in the centuries before Shakespeare’s life), English capitalisation practices were inconsistent, to say the least.

And, because Shakespeare’s work is on par with the KJV as a source of authority when it comes to formal and prestige English usage, the _always capitalise the first word of a sentence_ practice is regularly argued for with doctrinal zeal.

But there have long been exceptions, even among the true believers.

[Tussenvoegsel’s](https://en.wikipedia.org/wiki/Tussenvoegsel), for example.

Also names that begin with the _ﬀ_ (the digraph f). This glyph exists for two reasons. First, because written English in the Middle Ages didn’t have the _F_ glyph. In this case, strictly speaking, _ﬀ_ and _F_ are equivalent glyphs.

But _ﬀ_ is also used to spell Welsh names that begin with a fricative labiodental (represented by **f** in the IPA). This is because the _F_ glyph represents the **v** sound in written Welsh.

So, given exceptions existed even in the days when the practice was becoming the forcefully enforced norm, and even in the most formal and pompous of Englishes, there is no good argument for insisting that Proper Nouns denoting things — _iPad_ and _iPhone_ and _eBay_, for example — should be forced to take an initial capital when they are the first word of a sentence.

And, by simple extension, there is no good argument for mangling the name of a code object (eg `overrideDefaults`) just because it is the first word of a sentence.

You can make a case for re-writing a sentence to avoid having the argument arise at all. In this particular case, for example:

```
The <code>overrideDefaults</code> replace any of the defaults set by a previous call to the <code>overrideDefaults</code> function.
```
But that’s a style question and, speaking for myself, I don’t have a strong preference for either approach in this case.

Reply:
I changed the sentence to `The <code>overrideDefaults</code> method replaces any of the defaults set by a previous call to the <code>overrideDefaults</code> function.`

Reply:
Fascinating as always Brian. Yeah, I was probably only suggesting adding a word before it, not changing its capitalisation (as the function name is case-sensitive anyway). Your example of iPad and iPhone are interesting. I think I also go out of my way to put a word before them, usually. It's good to know that I belong to the elitist group of enforcing capitalisation in others based on an ignorance of context, history, and justification :P

Now, of course, it's all different in German again. I'm sure that's also a fascinating story as well. But try not to bite ...